core/src/main/scala/org/apache/spark/scheduler/local/LocalSchedulerBackend.scala

```diff
@@ -76,6 +81,9 @@ private[spark] class LocalEndpoint(
 
     case KillTask(taskId, interruptThread, reason) =>
       executor.killTask(taskId, interruptThread, reason)
+
+    case UpdateDelegationTokens(tokens) =>
```

Comment:
Just thinking out loud. Because in local version of Spark the executor, backend, and master all run in the same JVM it's not necessary to send the tokens to the `executorEndpoint` so it's fine as it.

Reply:
I thought about this, and seems it's the most easily approach to leave it here, since the `HadoopDelegationTokenManager `  would renew token through endpoint.